content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -8,7 +8,7 @@ tocDepth: 2
 
 The [Prisma schema](./) includes mechanisms that allow you to define names of certain database objects. You can:
 
-- [Map model and field names to different collection/table and field/column names in the underlying database](#mapping-collectiontable-and-fieldcolumn-names)
```

Comment:
unnecessary duplication that leads to line break which makes far harder to read and take in